src/convergents.jl

```diff
@@ -0,0 +1,22 @@
+immutable ConvergentIterator{T<:Integer,CF<:ContinuedFraction}
```

Comment:
You might not always want them to be the same, e.g. you might want `T` to be `BigInt`s (to avoid overflow), whereas the iterator might just give you `Int`s.